spec/features/peeps_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative '../web_helper.rb'
+require 'timecop'
+
+feature 'Viewing peeps' do 
+  include Helpers
```

Comment:
Could just add this into spec_helper as a require without it being a class - would DRY things up a bit

Reply:
Though now I read the review notes they use this module Helpers - not sure it needs to be called every time though (or maybe I'm wrong hence why you've done it..)